### PR TITLE
SDNTB-205 Latest translation changes

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: scripts/superdesk/views/confirmation-modal.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="modal-header">
-    <h3 ng-bind-html="headerText"></h3>
+    <h3></h3>{{ headerText | translate }}
```
Comment (petrjasek) [inline scripts/superdesk/views/confirmation-modal.html]: shouldn't this be inside those `h3` tags?

**Comment by petrjasek** [comment]
:+1:

File: scripts/superdesk/views/confirmation-modal.html (hunk 1 above)
Comment (vladnicoara) [inline scripts/superdesk/views/confirmation-modal.html]: You are right. Thank you @petrjasek . Fixed now

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**